Lib/fontTools/varLib/interpolatable.py

```diff
@@ -55,6 +64,21 @@ def addComponent(self, glyphName, transformation):
         self.value[-1].addComponent(glyphName, transformation)
 
 
+class RecordingPointPen(BasePen):
```

Comment:
We already have a RecordingPointPen:
https://github.com/fonttools/fonttools/blob/4bb0e776a72b508856f91ac40ea150772a3d4a80/Lib/fontTools/pens/recordingPen.py#L97

Reply:
Thanks.

Reply:
Hum. I renamed mine to `SimpleRecordingPointPen` since handling the full version was cluttering the code when I tried now.